guardian/editorial-viewer
# Migrate from deprecated node-sass to sass

## What does this change?
node-sass has been deprecated, and it doesn't seem any versions of it are compatible with ARM (now the architecture of most dev machines). This PR replaces the node-sass library with sass.

The new library requires a couple of syntax changes.

## How to test
See if you can run this tool locally on your ARM machine.

I have [deployed](https://riffraff.gutools.co.uk/deployment/view/99aa0896-ce9b-42dd-8d09-f439ffeb4326) this branch to [CODE](https://viewer.code.dev-gutools.co.uk/). Does it work as before?
